analyticsdataserver/router.py

```diff
@@ -7,8 +21,11 @@ def db_for_read(self, model, **hints):  # pylint: disable=unused-argument
         return self._get_database(model._meta.app_label)
 
     def _get_database(self, app_label):
-        if app_label in ('v0', 'enterprise_data'):
-            return getattr(settings, 'ANALYTICS_DATABASE', 'default')
+        if app_label in ('v1', 'v0', 'enterprise_data'):
+            if hasattr(request_cfg, 'database'):
+                return request_cfg.database
+            else:
+                return getattr(settings, 'ANALYTICS_DATABASE', 'default')
```

Comment:
this is interesting but I actually think for the shell we want v1 to still use v0 so that there is identical v1 data rather than empty

then as we actually make v1 calls they can connect up to v1 DB in a less exciting way

Reply:
you have a good point we don't need this for ALL v0 urls. I think my brain got sidetracked down the wrong path here.